index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
-
-  add(item) {}
-
-  get(pos) {}
-
-  max() {}
-
-  min() {}
-
-  sum() {}
-
-  avg() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a, b){return a-b});
+    this.length = this.items.length;
+  }
+  get(pos) {
+    if(pos > this.items.length) {
+      throw new Error('OutOfBounds');
+    } else{
+      return this.items[pos];
+    }
+  }
+  max() {
+    if(this.items.length === 0) {
```

Comment:
Daina what if the highest number was in a random position? Take a look at the Math.max, or another solution could be by sorting the array and returning the first or the last depending of the sort! 👍